torchtext/transforms.py

```diff
@@ -485,3 +485,17 @@ def bytes_to_unicode():
             n += 1
     cs = [chr(n) for n in cs]
     return dict(zip(bs, cs))
+
+
+class Sequential(torch.nn.Sequential):
```

Comment:
Does it make sense to break this PR into two separate commits -- one which brings `Sequential` out of experimental and another which migrates `XLMRTransform`?

Reply:
Indeed yes :). I have another PR to cover `Sequential` https://github.com/pytorch/text/pull/1481. Since that PR hasn't landed yet, I just created this PR out of the feature branch. I wonder if there are better ways to handle this? @mthrok any thoughts on creating new PR on existing PR(s)?

Reply:
One idea would be to branch off the feature branch where Sequential is un-experimentalized so you have the same history from this PR. I think if you do that, once that is merged you can just rebase main onto this branch for linear history.

Reply:
> Indeed yes :). I have another PR to cover `Sequential` #1481. Since that PR hasn't landed yet, I just created this PR out of the feature branch. I wonder if there are better ways to handle this? @mthrok any thoughts on creating new PR on existing PR(s)?

If the question is what I think it is, then I do not have a good solution. I got used to deal with multiple of PRs and commits. (at the moment I am juggling about 5 to 6 branches covering 3 different, independent aspects of one project.) Maybe gh-stack might help, but I have not tried yet. https://github.com/ezyang/ghstack

Reply:
Here is a nice blog post by the author of ghstack: https://0xc0d1.com/blog/git-stack/